data/tools.yml

```diff
@@ -27,6 +27,21 @@ tools:
       the same, `mdx` being the deprecate name, kept for now for compatibility.
     lifecycle: incubate
 
+  - name: OCaml Platform Installer
```

Comment:
I'm not completely sure the installer should be a Platform tool. Can we not add it for now and re-evaluate later?

Reply:
ok